### DISCUSSION TITLE
Package name in AUR

### DISCUSSION THREAD

**Discussion by mdujava** [beaker-project/beaker#142] [Q&A]
Hi all,

As I mentioned last meeting about publishing packages in Arch AUR.

If we want to keep same package names in arch as in Fedora (beaker-common, beaker-client), we would need to have name `beaker` for aur PKGBUILD, so we can have split package similar how spec file is done for RHEL distros. 

Currently name `beaker` is taken in AUR repositories. It's packaged beakerbrowser.com. It's also packaged under 5 other names and no much updates are happening there.

Question is should I push for marking those packages as Orphaned, and try to delete them (normal procedure when maintainer is not responding, and no one gets ownership) so we can use that package name (I think it's possible as we would not have beaker package on it's own, so old users would not be able to update their package)
Or should we find new prefix for a split package like `beaker-project-`.

What are your thoughts?

Thanks,
Matej

**Comment by mdujava** [comment]
all already existing packages renamed with prefix beaker-browser. Last request for beaker -> beaker-browser is pending. I will update here after beaker and beaker-git will be pushed by me.

**Reply by StykMartin** [reply]
🥳

**Reply by mdujava** [reply]
We are alive :) https://aur.archlinux.org/pkgbase/beaker/
